src/main/java/org/isf/medicalstockward/gui/WardPharmacy.java

```diff
@@ -576,33 +576,28 @@ public Component getTableCellRendererComponent(JTable table, Object value, boole
 		}
 	}
 
-	private MedicalWard showLotDetail(ArrayList<MedicalWard> drug, String me) {
+	private void showLotDetail(ArrayList<MedicalWard> drug, String me) {
 		ArrayList<MedicalWard> dr = new ArrayList<>();
```

Comment:
maybe for sake of readibility we can change `dr` to `medicalWardList`

Reply:
done